### PR TITLE
Receipt paper wallet

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Private key printing](https://trello.com/c/A9y5mQE1/136-private-key-printing)

File: lib/mocks/scanner.js [hunk 1]
```diff
@@ -63,6 +64,18 @@ function scanMainQR (cryptoCode, callback) {
   }, cbTimeout)
 }
 
+function scanPK (callback) {
+  _cancelCb = callback
+  handle = setTimeout(function () {
+    var resultStr = mockData.pk
+    try {
+      callback(null, resultStr)
```
Comment (joshmh) [inline lib/mocks/scanner.js]: Why is this in a ``try`` block?

File: lib/scanner.js [hunk 2]
```diff
@@ -171,6 +172,20 @@ function scanMainQR (cryptoCode, callback) {
...
 }
 
+function scanPK (callback) {
+  scanQR(function (err, result) {
+    if (err) return callback(err)
+    if (!result) return callback(null, null)
+
+    var resultStr = result.toString()
+    try {
+      callback(null, resultStr)
```
Comment (joshmh) [inline lib/scanner.js]: Why is this in a ``try`` block?

Comment (RafaelTaranto) [reply lib/scanner.js]: leftovers from copy pasting :man_facepalming:

**Pull request merged**